backend/src/apiserver/model/experiment.go

```diff
@@ -1,14 +1,29 @@
+// Copyright 2018 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package model
 
 type Experiment struct {
-	UUID           string `gorm:"column:UUID; not null; primary_key"`
-	Name           string `gorm:"column:Name; not null; unique_index:idx_name_namespace"`
-	Description    string `gorm:"column:Description; not null"`
-	CreatedAtInSec int64  `gorm:"column:CreatedAtInSec; not null"`
-	Namespace      string `gorm:"column:Namespace; not null; unique_index:idx_name_namespace"`
-	StorageState   string `gorm:"column:StorageState; not null;"`
+	UUID           string       `gorm:"column:UUID; not null; primary_key;"`
```

Comment:
Make the format consistent, either have the tab between type and gorm info for all models, or just use space.

Reply:
Maybe I am not seeing something. Could you please elaborate? The code has been linted and the format is consistent.